docs/references/faq.md

```diff
@@ -279,3 +279,19 @@ error\...
 2.  Next, go to the \\backend sub-folder and add this line to the
     **requirements-thirdparty.txt** file: `protobuf==3.20.1`.
 3.  Rebuild the plugin.
+
+## _"My plugin doesn't need UI"_
+
+Just simply delete the UI folder. For new plugins, [use --omit-ui](ayx-plugin-cli.md#create-ayx-plugin)
```

Comment:
TW review:

If your plugin doesn't need a UI, you can simply delete the UI folder. For new plugins, [use --omit-ui](ayx-plugin-cli.md#create-ayx-plugin).